server.js

```diff
@@ -27,6 +27,11 @@ app.use(session({
 app.use(passport.initialize());
 app.use(passport.session());
 
+app.get('*', function(req, res, next) {
```

Comment:
With this bit of code, we could write something like this in our views to check if we are signed in or not:

```ejs
<% if (user) { %>
  <h1>Log Out!</h1>
<% } else { %>
  <h1>Sign In!</h1>
<% } %>
```